File: LocadoraAPI/LocadoraAPI/Models/Veiculo.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace LocadoraAPI.Models
{
    public partial class Veiculo
    {
        public Veiculo()
        {
            Locacao = new HashSet<Locacao>();
        }

        public int IdVeiculo { get; set; }
        public string DsVeiculo { get; set; }
        public string CorVeiculo { get; set; }
        public string PlacaVeiculo { get; set; }
        public int AnoVeiculo { get; set; }
        public double PrecoVeiculo { get; set; }
        public bool? Alugado { get; set; }
        public int IdModelo { get; set; }

        public Modelo Modelo { get; set; }

        [JsonIgnore]
        public ICollection<Locacao> Locacao { get; set; }
    }
}

File: Locadora/src/app/cadastros/model/veiculo.ts
import { Modelo } from './modelo';

export class Veiculo {
    idVeiculo: number;
    dsVeiculo: string;
    corVeiculo: string;
    placaVeiculo: string;
    anoVeiculo: number;
    precoVeiculo: number; //no banco é double
    alugado: boolean;
    idModelo: Modelo;
}
